### PR TITLE
Create block `Our team`

### LINKED ISSUE #478: [About Us] TeamMembers

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## Related issue
Closes #478

## What does this PR do?
- Creation of the `Our team` block
- Addition of the story inside storybook
- Addition of basic tests to ensure all the blocks are present inside the `about us` page

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/utils/shuffle.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  for (let i = shuffleArray.length - 1; i > 0; i--) {
+    let j = Math.floor(Math.random() * (i + 1))
+    let temp = shuffleArray[i]
+    shuffleArray[i] = shuffleArray[j]
+    shuffleArray[j] = temp
+  }
+  return shuffleArray
+}
+
+export default shuffle
```
Comment (bidoubiwa) [inline src/utils/shuffle.js:12]:
```suggestion
const shuffle = array => {
  let shuffleArray = array
  for (let i = shuffleArray.length - 1; i > 0; i--) {
    let j = Math.floor(Math.random() * (i + 1))
    let temp = shuffleArray[i]
    shuffleArray[i] = shuffleArray[j]
    shuffleArray[j] = temp
  }
  return shuffleArray
}

export default shuffle
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
LGTM 🔥🔥🔥🔥

Just a small change on the shuffle function

**Review by bidoubiwa — APPROVED** [review]
🔥

**Comment by mdubus** [comment]
bors merge

**Pull request merged**